gov/agendas/tracker.go

```diff
@@ -191,7 +191,7 @@ func NewVoteTracker(params *chaincfg.Params, node VoteDataSource, counter voteCo
 func (tracker *VoteTracker) Refresh() {
 	voteInfo, err := tracker.refreshRCI()
 	if err != nil {
-		log.Errorf("VoteTracker.Refresh -> refreshRCI: %v")
+		log.Errorf("VoteTracker.Refresh -> refreshRCI: %v", err)
```

Comment:
@buck54321 I noticed these missing args, so I plugged them in.

I'm seeing this in the dcrdata logs:

```
[ERR] AGDB: VoteTracker.Refresh -> refreshRCI: refreshRCI: Vote information not found: -32603: stake version 8 does not exist
```

Reply:
Whoops. I missed this PR. I'll get you a review and look into the error.

Reply:
Thanks.  I was ready to merge this PR (unrelated to this error message though) when I hit a problem in testing on alpha.  For the life of me I cannot understand why it works with my local client, but not other socket.io clients from third parties hitting alpha.

Reply:
Sorry to change up the PR so much, but I discovered a few issues.